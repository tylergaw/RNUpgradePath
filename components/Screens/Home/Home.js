import { StyleSheet, View } from "react-native";

import { Button } from "Patterns";
import React from "react";

const Home = () => (
  <View style={styles.container}>
    <Button
      style={styles.button}
      title="henlo"
      onPress={() => console.log("Dogs are cool")}
    />
  </View>
);

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: "center",
    alignItems: "center",
    backgroundColor: "#F5FCFF"
  },
  button: {
    borderWidth: 2,
    borderRadius: 6,
    borderColor: "#252525",
    padding: 10
  }
});

export default Home;
