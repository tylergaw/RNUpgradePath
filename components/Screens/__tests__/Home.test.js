import Home from "../Home";
import React from "react";
import renderer from "react-test-renderer";

describe("Home", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
