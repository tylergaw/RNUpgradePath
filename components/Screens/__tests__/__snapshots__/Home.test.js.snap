// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "borderColor": "#252525",
        "borderRadius": 6,
        "borderWidth": 2,
        "opacity": 1,
        "padding": 10,
      }
    }
  >
    <Text>
      henlo
    </Text>
  </View>
</View>
`;
